app/Console/Commands/ScrapMoneyBox.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Panther\Client;
+
+class ScrapMoneyBox extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'scrap:moneybox';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Scrap the current moneybox value and save to DB';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $moneyboxId = env('MONEYBOX_ID');
```

Comment:
Lepszą praktyką jest trzymanie takich rzeczy w configu, bo potem wchodzi cacheowanie i może nie łapać wartości (`env('')` zwróci pusty string)

Zrób nowy plik w configu - `wosp.php` i tam daj `env`
https://laravel.com/docs/9.x/configuration#configuration-caching


Dalsze czytanie:
https://laravel.com/docs/9.x/configuration#configuration-caching


```suggestion
        $moneyboxId = config('wosp.moneybox.id');
```